README.md

```diff
@@ -55,9 +55,109 @@ $ yarn audit
+
+***The stacks to be deployed are:***
+1. alerting-integration
+2. api-gateway-logs
+3. certificate-expiry
+4. vpc
+5. lambda-audit-hook
+6. checkov-hook
+7. infra-audit-hook
+
+#### Stack Orchestation Tool Prerequisites
+
+- [aws-vault](https://github.com/99designs/aws-vault)
+- [jq](https://formulae.brew.sh/formula/jq)
+
+#### ⚡ Prior to deploying check the latest version is being referenced which can be found in the [CHANGELOG](https://github.com/govuk-one-login/devplatform-deploy/tree/main)
+
+#### Steps:
+
+```shell
+$ cd stack-orchestation
+$ aws configure sso
+$ aws sso login --profile di-account-intervention-admin-324281879537
+$ sh production_bootstrap.sh
+```
+The bootstrap script would deploy all base Cloudformation stacks required for account set up.
+
+### 2. Deploy ais-infra stacks
+#### Clone the [ais-infra](https://github.com/govuk-one-login/ais-infra/blob/3046cb392b657ded2b652a52f0652a6ccd4d8630/) repo
+```shell
+git clone git@github.com:govuk-one-login/ais-infra.git
+cd ais-infra
+```
+##### ⚠️ Setup pre-commit
+```shell
+$ pre-commit install -f
+```
+
+#### Deploy the following stacks as ordered:
+1. `ais-infra-alerting` - [README.md](https://github.com/govuk-one-login/ais-infra/blob/0087789dc22a0abede2458a67106f7e45dff2b99/ais-infra-alerting/README.md)
+2. `ais-infra-common` - [README.md](https://github.com/govuk-one-login/ais-infra/blob/c3ce026093e69cbfdf4272b119a1fe8cb31ee4cf/ais-infra-common/README.md)
+3. `ais-dynatrace-metrics` - [README.md](https://github.com/govuk-one-login/ais-infra/blob/c3ce026093e69cbfdf4272b119a1fe8cb31ee4cf/ais-dynatrace-metrics/README.md)
 
+### 3. Setup secure pipelines
+
+#### Setup these 3 secure pipelines by using the Stack Orchestation tool provided in our [Stack-Orchestration](stack-orchestration) directory and deploy the [production_pipelines.sh](stack-orchestration%2Fproduction_pipelines.sh) script.
+
+1. `ais-core-pipeline` deploy our `ais-core` stack
+2. `ais-main-pipeline` deploys our `ais-main` stack
+3. `ais-alarm-pipeline` deploys our `ais-alarm` stack
+
+
+#### ⚡ Prior to deploying check the latest version of secure pipelines is being referenced in the [production_pipelines.sh](stack-orchestration%2Fproduction_pipelines.sh) script >  [**CHANGELOG**](https://github.com/govuk-one-login/devplatform-deploy/blob/main/sam-deploy-pipeline/CHANGELOG.md#added-32)
+
+#### Deployment Steps:
+
+```shell
+$ cd stack-orchestation
+$ aws configure sso
+$ aws sso login --profile di-account-intervention-admin-324281879537
+$ sh production_pipelines.sh
+```
+
+Once all production pipelines have been set up proceed to deploy the [staging_pipelines.sh](stack-orchestration%2Fstaging_pipelines.sh) script to allow promotion up to `production` account.
```

Comment:
Once all production pipelines have been set up, proceed to running the [staging_pipelines.sh](stack-orchestration%2Fstaging_pipelines.sh) script to allow promotion up to `production` account.